kong/plugins/prometheus/enterprise/exporter.lua

```diff
@@ -0,0 +1,94 @@
+local kong = kong
```

Comment:
question: do we plan to port this PR to EE as well? if so we can remove those EE code from here. and add them back in the EE sister PR.